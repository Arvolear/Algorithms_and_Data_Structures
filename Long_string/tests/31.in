91579328515528306508079273990810989013084
48479041140299608