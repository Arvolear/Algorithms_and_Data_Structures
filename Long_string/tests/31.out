91579328515528306508079322469852129312692
91579328515528306508079225511769848713476
4439678034705309799497585271038130533451226133071992071072
1889049914384555256105264